### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by jwintz**
Merge request for updated emacs-plus formula.

### CODE REVIEW

**Comment by bmag** [comment]
Can be squashed to a single commit

File: README.md [hunk 1]
```diff
@@ -157,6 +157,9 @@ providing Spacemacs icon.
 See [emacs-plus](https://github.com/d12frosted/homebrew-emacs-plus) for more
 information.
 
+*Note*:* if you are are willing your theme to match the title bar theme, please use the install toggle switch statement:
```
Comment (robbyoconnor) [inline README.md]: The english here is horrible...

**Comment by robbyoconnor** [comment]
Also @jwintz -- title your PR something that actually describes your change...

**Comment by jwintz** [comment]
Updated PR title, and fixed install command.

**Comment by jwintz** [comment]
Squashed into a single commit.

File: README.md (hunk 1 above)
Comment (jwintz) [inline README.md]: Why don't you propose a better rhetoric then :-)

**Comment by jwintz** [comment]
Here's the result for the record.

**Comment by syl20bnr** [comment]
Thank you ! 👍 
Cherry-picked into develop branch, you can safely delete your branch.

**Comment by jwintz** [comment]
Thank you :-)

**Comment by robbyoconnor** [comment]
@syl20bnr why'd you change it back?

**Comment by jwintz** [comment]
@robbyoconnor, do you mean the PR title ? Don't know why it got back to "Devel". Il'l change it again.

Anyway is my new sentence less horrible ? ;-) (Agree it was not beautiful English)

**Comment by robbyoconnor** [comment]
@jwintz yeah -- sorry if that came off a bit...abrasively.

**Pull request merged**